common.sh

```diff
@@ -119,8 +119,8 @@ fi
 
 if [ ! -d "$WORKING_DIR" ]; then
   echo "Creating Working Dir"
-  mkdir "$WORKING_DIR"
-  chown "${USER}:${USER}" "$WORKING_DIR"
+  sudo mkdir "$WORKING_DIR"
+  sudo chown "${USER}:${USER}" "$WORKING_DIR"
   chmod 755 "$WORKING_DIR"
```

Comment:
As mentioned in https://github.com/openshift-metalkube/dev-scripts/pull/205 _sudo_ is needed for _chmod_ as well
```suggestion
  sudo chmod 755 "$WORKING_DIR"
```